components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -136,6 +136,7 @@ public class EndpointUtil {
     private static final String PROP_GRANT_TYPE = "response_type";
     private static final String PROP_RESPONSE_TYPE = "response_type";
     private static final String PROP_SCOPE = "scope";
+    private static final String PROP_OIDC_SCOPE = "requested_oidc_scope";
```

Comment:
```suggestion
    private static final String PROP_OIDC_SCOPE = "requested_oidc_scopes";
```

Reply:
But other parameter keys are also in singular form although they got multiple values. ex: scope in line 138.

Reply:
These parameters(response_type, scope) are well defined in the spec (OAuth 2.0 spec). Since `requested_oidc_scope` is something we are introducing we can decide based on our context.